packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -140,15 +145,21 @@ const DropdownBase = kind({
 
 	propTypes: /** @lends moonstone/Dropdown.DropdownBase.prototype */ {
 		/**
-		 * The selection items to be displayed in the `DropdownList`.
-		 * Takes an array of strings and the strings will be used in
-		 * the generated components as the readable text.
+		 * The selection items to be displayed in the `Dropdown` when `open`.
 		 *
-		 * @type {String[]}
-		 * @required
+		 * Takes either an array of strings or an array of objects. When strings, the values will be
+		 * used in the generated components as the readable text. When objects, the properties will
+		 * be passed onto an `Item` component and a unique `key` property is required.
+		 *
+		 * @type {String[]|Array.<{key: (Number|String)}>}
 		 * @public
 		 */
-		children: PropTypes.node,
+		children: PropTypes.oneOfType([
+			PropTypes.arrayOf(PropTypes.string),
+			PropTypes.arrayOf(PropTypes.shape({
+				key: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired
```

Comment:
Should we also require a `children` prop of type `string` or `Component`?

Reply:
Yes. Probably for both this and ExpandableList